demisto_sdk/commands/update_xsoar_config_file/update_xsoar_config_file.py

```diff
@@ -131,7 +131,7 @@ def update_xsoar_config_data(self, section_name, data_to_update):
         """
         config_file_info = self.get_xsoar_config_data()
         if config_file_info.get(section_name):
-            config_file_info[section_name].append(data_to_update)
+            config_file_info[section_name].extend(data_to_update)
```

Comment:
is `data_to_update` always a list/tuple?

Reply:
list or dict, fixed...